backend/tests/v5/e2e/routes/teamspaces/projects/models/common/tickets.test.js

```diff
@@ -664,11 +671,11 @@ const testGetTicketList = () => {
 			const baseRouteParams = { key: users.tsAdmin.apiKey, modelType, projectId: project.id, model };
 
 			const checkTicketList = (ascending = true) => (tickets) => {
-				// check the list is sorted by created at, ascending order
+				// check the list is sorted by updated at, ascending order
```

Comment:
why did you need to change this?

Reply:
the test is actually passing UPDATED_AT

Reply:
by default the tickets should be sorted by createdAt in descending order unless it's specified by the query string?